quickstart.md

```diff
@@ -20,18 +20,42 @@ git clone git@github.com:<github_username>/i18n.git
 ```
 git remote add ja git@github.com:swcarpentry-ja/i18n.git
 ```
-5. Make changes to your local repo - make sure to commit often:
+5. Create new branch for your edit (separate editing branch will make things easier for Pull Request):
```

Comment:
I'm not a fan of excessive merge commits either but I don't know if we can really avoid them. I don't recommend people who are inexperienced with git to be handling branches and rebase. If we push rebased commits to a remote repo, it could be a problem for someone else.

I encountered a similar issue when resolving conflicts for PR https://github.com/swcarpentry-ja/i18n/pull/32 from @rkkmk. I originally made a few branch, pulled the user repo and submitted changes as another PR. Then I realised that this wasn't necessary. It is possible to pull commits from a user repo (their branch) and merge locally. Pushing to the organisation repo merges the PR via the command-line. I recommend this workflow to merge PRs. This way, only admins are handling branches and merges where things get more difficult with Git.

It should be possible to submit a PR without doing this. I want to lower the barrier for entry as much as possible. Creating a fork or pulling to a local repo creates a different branch specific that machine/account already. It should be possible to merge. I am willing to merge PRs and resolve conflicts where needed. However it would help if member submitting pull requests would pull from the organisation repo and merge changes _before_ pushing to their account and submitting a pull request. They should also be pulling from the organisation repo _regularly_ to ensure that they're working on the latest version (to reduce conflicts with things others have already changed).